openedx/core/djangoapps/config_model_utils/models.py

```diff
@@ -167,13 +227,13 @@ def current(cls, site=None, org=None, org_course=None, course_key=None):  # pyli
             raise ValueError("Only one of site, org, org_course, and course can be specified")
 
         if org_course is None and course_key is not None:
-            org_course = cls._org_course_from_course_key(course_key)
+            org, org_course = _org_course_from_context_key(course_key)
```

Comment:
My assumption was that a CourseKey is a subset/type of context key so this would be OK, but I get your point from the previous comment, and will change accordingly.